tests/test_plotting.py

```diff
@@ -0,0 +1,38 @@
+import shutil
+from pathlib import Path
+from nesta_ds_utils import plotting
+import pandas as pd
+import altair as alt
+import pytest
+
+
+def test_save_altair():
+    """Test that that figures saved exist."""
+    fig = alt.Chart(pd.DataFrame()).mark_bar()
+    path = "tests/temp/"
+    plotting.save(fig, "test_fig", path)
+    assert all(
```

Comment:
I'm not a testing expert, but I think better practice would be to have multiple separate asserts. This way you know if an individual format failed.